crates/ruff_python_formatter/src/other/match_case.rs

```diff
@@ -24,7 +24,17 @@ impl FormatNodeRule<MatchCase> for FormatMatchCase {
             [
                 text("case"),
                 space(),
-                not_yet_implemented_custom_text("NOT_YET_IMPLEMENTED_Pattern"),
+                format_with(|f: &mut PyFormatter| {
```

Comment:
This is intentionally ugly to make sure we remove we don't forget to remove the `comment.mark_formatted` when replacing this with `pattern.format()`